src/main/resources/cz/brmlab/yodaqa/analysis/rdf/Mbprop.txt

```diff
@@ -0,0 +1,51 @@
+6.478921336989754143e-02 -1.572191018898824039e-02 1.357541785864419259e-01 6.197760020473357112e-02 -1.444565824322490144e-01 -3.173638599022809981e-01 1.478570290364157602e-01 -4.856656452107218497e-02 1.935289609928618482e-01 -3.052286513235973400e-02 -1.621755634230770404e-01 1.757835127771281702e-01 -9.751116032475977413e-02 -7.869799945316685585e-02 1.498097831504699440e-02 1.010940708298489754e-01 -7.509524079728964463e-02 2.384814117417059787e-01 3.292959231849696966e-01 -2.504127929101749306e-01 1.852284373308361043e-01 1.442353404702851061e-01 -2.969319351500072091e-01 -2.231603107790339907e-02 -8.153610917523854140e-02 -1.304090610640723791e-01 4.861604506830530942e-02 -4.430417777449092986e-02 1.422131788629214799e-01 -7.025973936849949641e-02 -1.907153599196960425e-01 3.002919329370101198e-01 7.040665099596032928e-02 -4.567225196175795504e-02 -1.559464323246115558e-01 1.260952560944670064e-01 -1.710434670118842204e-01 -2.186184977782067718e-01 2.304735026750457522e-02 -1.171818411594163084e-01 -4.823173311596493096e-02 1.320226879541117893e-01 1.951184581715055821e-01 1.607350202889098745e-01 2.475512801426262566e-01 1.961365882619048873e-01 -1.279237801352386683e-01 -1.669237992452196617e-01 1.191133237513752469e-01 2.113182161974995477e-02
```

Comment:
It would be great if at the beginning of the file, we could have a few comment-like lines that describe details about the training process (and performance of the model on the selection subtask), like we have for the models.